src/pages/routes-and-stops/routes-and-stops.component.ts

```diff
@@ -72,6 +74,19 @@ export class RoutesAndStopsComponent {
       }
     }
   }
+  
+  showToast(message: string): void {
+    if (this.toast) {
+      this.toast.dismissAll();
+    }
+  
+  this.toast = this.toastCtrl.create({
```

Comment:
Okay, figured it out. It's partially my fault, partially vim's. Vim isn't recognizing typescript syntax, so I'm left with weird indentation problems that I don't notice.